components/bsx/Offer/OfferTable.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <div>
-    <Pagination
-      :total="total"
+  <div class="offer-table-container">
+    <b-table
+      :data="displayOffers(offers)"
```

Comment:
where does this `displayOffers` come from?

Reply:
It comes from `OfferMixin`

Reply:
Alright, I hadn't pulled the latest version